pallets/protos/src/lib.rs

```diff
@@ -251,6 +251,7 @@ pub mod pallet {
 		StakeLocked,
 		/// Cannot find FRAG link to use as stake funds
 		NoFragLink,
+		OwnerNotFound,
```

Comment:
hmm we generally use those errors for extrinsics. Maybe we need a new struct?

Reply:
Oh, this can be removed actually. I am not using it.